config/src/main/kotlin/maru/config/Config.kt

```diff
@@ -191,6 +190,8 @@ data class SyncingConfig(
     val blocksBatchSize: UInt = 10u,
     val blocksParallelism: UInt = 1u,
     val maxRetries: UInt = 5u,
+    val pauseBetweenAttempts: Duration = 1.seconds,
```

Comment:
Maybe we can reuse the concept of backoff ?

```suggestion
    val backoffDelay: Duration = 1.seconds,
```

Reply:
Yes, I renamed it